src/main/java/blockchain/ContractCV.java

```diff
@@ -0,0 +1,43 @@
+package blockchain;
+
+import evm.Contract;
+import evm.ContractTransaction;
+import skipGraph.NodeInfo;
+import java.rmi.RemoteException;
+
+class ContractCV extends CorrectnessVerifier {
```

Comment:
the class requires separate unit tests and java docs. Regarding the tests, we should have a separate test verifying the correctness of a _correct_ contract, and another one verifying the correctness of an _incorrect_ one. So we can cover both happy and unhappy paths.

Reply:
Check here db20b90eb21e9a4a55662adbd4a69ac06cd5d83a